### DISCUSSION TITLE
Update the website so it looks pretty

### DISCUSSION THREAD

**Discussion by mmnvb** [multitheftauto/mtasa-blue#3361] [Ideas] [answered]
Hi there, I noticed your project recently and it is amazing! I was trying to download some resources for my server and noticed that the website is looking old. Let's update it and I can help with that too. It would be great to have something like curseforge for MTA

**Comment by Lime-blur** [comment]
> Hi there, I noticed your project recently and it is amazing! I was trying to download some resources for my server and noticed that the website is looking old. Let's update it and I can help with that too. It would be great to have something like curseforge for MTA

Yes, as far as I know, the site will change in the future. For me personally, the main thing is that the RSS feed doesn't change or remove. Or at least there would be an alternative API since some developers like me depend on it.

In fact, this is a huge job that requires a lot of work, and, most importantly, all this work doesn't improve the MTA:SA core itself. Therefore, such a task is not the most priority at the moment, I think.

**Comment by alandmine** [comment]
it is so ugly it makes me want to throw up

**Comment by qaisjp** [comment, marked as answer]
Someday!
